packages/blade/src/components/IconButton/IconButton.stories.internal.tsx

```diff
@@ -0,0 +1,66 @@
+import type { ComponentStory, Meta } from '@storybook/react';
+import { Title, Subtitle, Primary, ArgsTable, Stories, PRIMARY_STORY } from '@storybook/addon-docs';
+import type { ReactElement } from 'react';
+import { Highlight } from '@storybook/design-system';
+
+import type { IconButtonProps } from './IconButton';
+import IconButtonComponent from './IconButton';
+import iconMap from '~components/Icons/iconMap';
+
+const Page = (): ReactElement => {
+  return (
+    <>
+      <Title />
+      <Subtitle>
+        Useful for making clickable icons. For example - close button for modals, inputs, etc.
+      </Subtitle>
+      <Title>Usage</Title>
+      <Highlight language="tsx">{`import { IconButton } from '@razorpay/blade/components' \nimport type { IconButtonProps } from '@razorpay/blade/components'`}</Highlight>
+      <Title>Example</Title>
+      <Subtitle>
+        You can change the properties of this button using the controls in the table below.
+      </Subtitle>
+      <Primary />
+      <Title>Properties</Title>
+      <ArgsTable story={PRIMARY_STORY} />
+      <Stories />
+    </>
+  );
+};
+
+const meta: Meta<IconButtonProps> = {
+  title: 'Components/IconButton (Internal)',
+  component: IconButtonComponent,
+  args: {
+    size: 'medium',
+    contrast: 'low',
+    accessibilityLabel: 'Close',
+  },
+  argTypes: {
+    onClick: { action: 'onClick' },
```

Comment:
I don't think we need to specify this. It should assume all onClicks as actions, right?

Reply:
Right, can be skipped, thought to make it explicit

Reply:
So it turns out without this, the click action doesn't work on android. Confirmed for buttons too. Works fine on ios. So we'll need these.